frontend/src/metabase/user/components/LoginHistoryList.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+
+import LoginHistory from "metabase/entities/loginHistory";
+
+const LoginHistoryItemRow = item => {
+  return (
+    <tr>
+      <td>
+        <tt>{item.timestamp}</tt>
+      </td>
+      <td>{item.location && item.location.description}</td>
+      <td>
+        <tt>{item.device_description}</tt>
+      </td>
+      <td>{item.active ? "✅" : "❌"}</td>
+    </tr>
+  );
+};
+
+@LoginHistory.loadList()
+export default class LoginHistoryList extends Component {
```

Comment:
Might consider making this a pure function instead of a class.

Reply:
would that still work with the `@LoginHistory.loadList()` decorator?